app/javascript/articles/Article.jsx

```diff
@@ -3,140 +3,93 @@ import PropTypes from 'prop-types';
 import { articlePropTypes } from '../src/components/common-prop-types/article-prop-types';
 import {
   CommentsCount,
+  CommentsList,
+  Cover,
   ContentTitle,
-  OrganizationHeadline,
-  PublishDate,
-  ReadingTime,
+  Author,
+  OverflowNavigation,
   SaveButton,
   SearchSnippet,
   TagList,
   ReactionsCount,
 } from './components';
 import { PodcastArticle } from './PodcastArticle';
 
-/* global timeAgo */
-
 export const Article = ({
   article,
   currentTag,
+  isFeatured,
   isBookmarked,
-  reactionsIcon,
-  commentsIcon,
-  videoIcon,
+  // videoIcon,
   bookmarkClick,
 }) => {
   if (article && article.type_of === 'podcast_episodes') {
     return <PodcastArticle article={article} />;
   }
 
-  const timeAgoIndicator = timeAgo({
-    oldTimeInSeconds: article.published_at_int,
-    formatter: x => x,
-  })
-
   return (
     <div
-      className="single-article single-article-small-pic"
+      className={`crayons-story ${isFeatured && ("crayons-story--featured")}`}
+      id={isFeatured && ("featured-story-marker")}
+      data-featured-article="TODO"
```

Comment:
I'm not really sure how to put these two lines under IF statement so it's only for featured articles..

Also re TODO part: https://github.com/thepracticaldev/dev.to/blob/5f322210ce1b44df0cb7bc0af38fb6fd053869a3/app/javascript/articles/FeaturedArticle.jsx#L22-L23